packages/terra-tabs/src/_TabMenu.jsx

```diff
@@ -76,6 +76,8 @@ class TabMenu extends React.Component {
 
   wrapOnClick(child) {
     return (event) => {
+      event.stopPropagation();
```

Comment:
Essentially, when we click on a tab, the new bounding rectangle is bubbling the click up to the menu click as well, which keeps the menu bar open.

Reply:
Is this something you found with the wdio screenshots?? Or in other test instances?

Reply:
I found it through the wdio screenshots, but then tested it in a browser and saw the same behavior.